package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "assignment",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.10",
+    "@testing-library/react": "^11.2.6",
+    "@testing-library/user-event": "^12.8.3",
+    "axios": "^0.21.1",
+    "check-prop-types": "^1.1.2",
```

Comment:
use prop types check in each component.
1) You have to add propTypes as well as default value.

Reply:
2. There should be some required props too which should be there in the propTypes check